src/vsphere_cpi/lib/cloud/vsphere/nsxt_provider.rb

```diff
@@ -303,15 +304,31 @@ def retrieve_nsgroups(nsgroup_names)
 
     def logical_ports(vm)
       Bosh::Retryable.new(
-        tries: @max_tries,
-        sleep: ->(try_count, retry_exception) { @sleep_time },
+        tries: [@max_tries, NSXT_MULTI_VM_COPY_RETRY].max,
```

Comment:
Can you just use nsxt values here instead and on  next line also?
If @max_tries and @sleep_time are not being used anywhere you could remove them